src/go/metrics/sender.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"context"
+
+	io_prometheus_client "github.com/prometheus/client_model/go"
+
+	"github.com/go-openapi/swag"
```

Comment:
can we avoid the use of swag?

also, protobuf also includes pointer-gen methods like `proto.String`